src/views/pages/login/Login.js

```diff
@@ -16,12 +16,12 @@ const Login = ({ authenticate }) => {
   }, [authenticate]);
 
   return (
-    <div className="container fluid">
+    <div className="container-fluid">
       <div className={styled["landing-page-container"]}>
         <div className={styled.content__wrapper}>
           <header className={styled.header}>
-            <h1 className="d-flex justify-content-center display-1">BUNDLY</h1>
-            <ul
+            <h1 className="d-flex justify-content-center display-3">BUNDLY</h1>
+            {/* <ul
```

Comment:
Remove the commented content?

Reply:
We might wanna add that later but yeah, the icons weren't displaying so we can add something new in the rework.